src/Symfony/Component/Security/Core/Security.php

```diff
@@ -57,4 +60,25 @@ public function getToken(): ?TokenInterface
     {
         return $this->container->get('security.token_storage')->getToken();
     }
+
+    /**
+     * Logout the current user automatically. Dispatch the logout event.
+     */
+    public function logout(): ?Response
+    {
+        $request = $this->container->get('request_stack')->getMainRequest();
+        $logoutEvent = new LogoutEvent($request, $this->container->get('security.token_storage')->getToken());
```

Comment:
We need a guard condition that checks if the LogoutEvent exists and throw an exception otherwise, because Security\Core does not depend on Security\Http. 
Btw we need to add `"symfony/security-http": "^5.4|^6.0"` as a dev requirement in security-core's composer.json in order to make tests pass.

Reply:
```php
throw new \LogicException('Security HTTP is missing. Try running "composer require symfony/security-http".');
```

Reply:
Done